config/galaxy.ini.sample

```diff
@@ -203,7 +203,8 @@ paste.app_factory = galaxy.web.buildapp:app_factory
 # Pass debug flag to conda commands.
 #conda_debug = False
 # conda channels to enable by default (http://conda.pydata.org/docs/custom-channels.html)
-#conda_ensure_channels = conda-forge,r,bioconda,iuc
+# the recommended channel order is the one from BioConda (https://github.com/bioconda/bioconda-recipes/blob/master/config.yml#L8)
+#conda_ensure_channels = bioconda,r,defaults,conda-forge,iuc
```

Comment:
@jmchilton @bgruening I don't think there is a rush to merge this or the need to backport it since channel order in Conda 3.19.3 matters only if 2 packages have the same version and build numbers. This changed in Conda 4.1.0, see http://conda.pydata.org/docs/channels.html